content/python/concepts/strings/terms/ljust/ljust.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.ljust()'
+Description: 'Left-aligns a string with a specified fill character'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The method **`.ljust()`** left aligns a string for the given length of characters. If the length of the string is less than the parameter given, then padding is added to the right of the given string. The user can specify a character to use as the padding. The number of padding characters added is the length of the string provided subtracted from the length specified in the method call.
+
+## Syntax
+
+```pseudo
+string.ljust(length, fillchar)
+```
+
+### Required Parameters
+
+- `string`: The string to be modified with padding.
+- `length`: An integer used to determine how many characters of padding will be added. The number of padding characters will be equal to `length` minus the length of `string`.
+
+### Optional Parameters
+
+- `fillchar`: A character to fill the missing space (to the right of the string). Default is " " (space).
+
+## Examples
+
+The default `fillchar` is a blank space. The original string is retrieved if the length is less than or equal to the given string's length.
+
+```python
+text = 'Hello Contributors'
+left_aligned_text = text.ljust(20, '!')
+print(left_aligned_text)
+```
+
+This code will result in the following output:
+
+```shell
+Hello Contributors!!
+```
+
+> **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with fillchar.
```

Comment:
```suggestion
> **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with the parameter `fillchar`.
```
Some readers might get confused if they directly read the sentence. I think it'll be a good practice to mention that it is a parameter.